tools/clang/test/HLSL/globallycoherent-errors.hlsl

```diff
@@ -0,0 +1,18 @@
+// RUN: %clang_cc1 -fsyntax-only -ffreestanding -verify %s
```

Comment:
hrm. Why didn't this appear like a move of any of the deleted copies? Why we had copies in the first place is another matter.